appcelerator/alloy
# [ALOY-1479] Regression: Model instance data binding no longer works

https://jira.appcelerator.org/browse/ALOY-1479

To verify:

1. Run `jake app:run dir=testing/ALOY-1474`, `jake app:run dir=testing/ALOY-1477` and `jake app:run dir=testing/ALOY-1479` and verify all specs pass and result is as `expected.png` in those test apps
2. Verify these 3 include all test cases mentioned in https://jira.appcelerator.org/browse/ALOY-1479?focusedCommentId=381041&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-381041
3. Check again :)
